src/components/Card/Card.tsx

```diff
@@ -15,20 +16,15 @@ export interface CardProps {
   name: string;
   price: number;
   onClick?: () => void;
-
 }
 
-export default function Card({
-  rank,
-  imageUrl,
-  brand,
-  name,
-  price,
-  onClick,
-}: CardProps) {
+export const Card = forwardRef<HTMLDivElement, CardProps>(function Card(
+  { rank, imageUrl, brand, name, price, onClick },
+  ref
+) {
   return (
-    <CardWrapper onClick={onClick} style={{ cursor: "pointer" }}>
-      <RankNumber>{rank}</RankNumber>
+    <CardWrapper ref={ref} onClick={onClick} style={{ cursor: "pointer" }}>
+      {rank !== -1 && <RankNumber>{rank}</RankNumber>}
```

Comment:
아래와 같이 범위로 조건을 보면 다른 유효하지 않은 값까지 커버할 수 있을 것 같습니다.

```suggestion
      {rank > 0 && <RankNumber>{rank}</RankNumber>}
```